cmat/clinvar_xml_io/clinvar_submitted_record.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from functools import cached_property
+
+from cmat.clinvar_xml_io import ClinVarRecord
+from cmat.clinvar_xml_io.xml_parsing import find_mandatory_unique_element
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class ClinVarSubmittedRecord(ClinVarRecord):
+    """
+    Submitted records (SCVs) are structured similarly to reference records (RCVs) with a few exceptions, though they
+    typically have fewer annotations - for example, variant coordinates, HGVS expressions or ontology mappings which are
+    added by curators.
+
+    SCVs also contain additional information about the actual submission, which we model in this class.
+    """
+
+    def __init__(self, record_xml, xsd_version, reference_record):
+        super().__init__(record_xml, xsd_version)
+        # Each SCV is associated with a single RCV
+        self.reference_record = reference_record
+
+    def __str__(self):
+        return f'ClinVarSubmittedRecord object with accession {self.accession}'
+
+    @property
+    def submission_date(self):
+        """Date of submission or when submission was last revised (for first submission, use created_date)."""
+        return find_mandatory_unique_element(self.record_xml, './ClinVarSubmissionID').attrib['submitterDate']
+
+    @property
+    def last_updated_date(self):
+        """Overrides parent definition, in SCV this date is in the accession element"""
+        return find_mandatory_unique_element(self.record_xml, './ClinVarAccession').attrib['DateUpdated']
+
+    @property
+    def created_date(self):
+        """Overrides parent definition, in SCV this date is in the accession element"""
+        return find_mandatory_unique_element(self.record_xml, './ClinVarAccession').attrib['DateCreated']
+
+    @property
+    def submitter(self):
+        """Name of the submitting organization."""
+        return find_mandatory_unique_element(self.record_xml, './ClinVarSubmissionID').attrib['submitter']
+
+    @property
+    def submitter_id(self):
+        """Numeric identifier associated with the submitting organization."""
+        return find_mandatory_unique_element(self.record_xml, './ClinVarAccession').attrib['OrgID']
+
+    @property
+    def submission_name(self):
+        """Name or identifier associated with the submission. This is optional."""
+        return self.record_xml.attrib.get('SubmissionName', None)
```

Comment:
This will be used for the gene-related condition filtering, confirmed it's present in that case